ch01.adoc

```diff
@@ -18,7 +18,35 @@ We have also striven to maximize conformance to the COARDS standard, that is, wh
 
 
 
-[[terminology, Section 1.2, "Terminology"]]
+[[terminology, Section 1.2, "Principles for design"]]
```

Comment:
Thanks, @JonathanGregory. Looks good. I think that perhaps you forgot to change the cross reference key from `terminology` to something more approriate

Reply:
Yes, quite right, thanks. Now corrected.